recipes/pyFAI/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install
```

Comment:
Delete this file since it's not necessary given the `script` section in the `meta.yaml` file.